src/terra-dev-site/doc/application/how-to.2/theme-strategy.doc.mdx

```diff
@@ -0,0 +1,166 @@
+import ExampleComponent from './example/ExampleComponent?dev-site-example';
+
+# Theming Guide
+
+Welcome to the Terra theming strategy guide. This guide will walk teams through the recommended approach for theming custom components.
+
+## Summary
+
+Terra supports a default, clinical-lowlight-theme, and orion-fusion-theme. This guide is for theming custom components. Custom components are any non Terra-UI component created by teams that need to be integrated into an application using a supported Terra theme. Teams should not create new themes or theme Terra components.
+
+## Getting Started
+
+To get started, check out the [terra-application](http://engineering.cerner.com/terra-application/components/terra-application/application/reference/components/application-base) documentation. For this guide, it is expected that teams are utilizing an application framework that defines a theme provider. This guide will walk through theming an example component.
```

Comment:
maybe a little re-organizing, you start off linking to terra-apps' docs but it might be helpful to mention this talks shows the theming setup is provided by base.

For theming applications, it is expected .... This guide is assuming you are using terra-application's Base component as the application framework, which sets these up for you. This guide...

Reply:
I did a little rephrasing, open to more suggestions:

https://github.com/cerner/terra-application/pull/25/commits/2a93997b6888d4c01d6be725178806bdd92ac386